code/__defines/misc.dm

```diff
@@ -1,4 +1,8 @@
 
+//Fullscreen overlay resolution in tiles.
+#define FULLSCREEN_OVERLAY_RESOLUTION_X 15
```

Comment:
this seems bad, these should be calculated from client's viewsize not hardcoded

Reply:
> this seems bad, these should be calculated from client's viewsize not hardcoded

Deleted